astronomer/providers/apache/hive/example_dags/example_hive.py

```diff
@@ -29,6 +32,7 @@
 
 PEM_FILENAME = os.environ.get("PEM_FILENAME", "providers_team_keypair")
 PRIVATE_KEY = Variable.get("providers_team_keypair")
+HIVE_SCHEMA = os.getenv("HIVE_SCHEMA", "default")
```

Comment:
Should we standardize using `os.environ.get` and `os.getenv`

Reply:
Agree, currently this is not uniform across the repo. though both do not have much differences but we should keep it uniform. What should I use `os.environ.get` or `os.getenv` ?

Reply:
Both are same tbh so I have no preference :)

Reply:
Ok, I'll update everywhere to use `os.getenv` . Since, this is not related to this PR I have filed another PR for this https://github.com/astronomer/astronomer-providers/pull/308